### ISSUE TITLE
Why are images that are not in the same folder not recognized?

### ISSUE THREAD

**Issue by ooker777** [webpro/reveal-md#488]
It seems that only images in the same folder are recognized. If it's in the parent then it won't work. Is this a conscious choice or you don't have the time to work on it?

**Comment by webpro** [comment]
It was a conscious choice to keep things simple. Assets and images should be in the same or sub folders.

**Comment by webpro** [comment]
The challenge was that reveal-md is basically a simple http server that serves static assets and also allows to export it all as a stand-alone HTML website.

Feel free to take a stab at it though, I'm happy to review PRs.

**Comment by ooker777** [comment]
I'm still not sure what simplicity this achieves. Is it for the users or for the devs? I'm not working on it yet, but I imagine it should be no problem for the server to get files from parents?

**Comment by ooker777** [comment]
Yeah I understand that. I just want to know why you make that decision, that's all. Not intend to ask you to work on it :)

**Comment by webpro** [comment]
> [!WARNING]
>
> reveal-md is no longer in active development.

No new features will be added. Pull requests for bug and documentation fixes will be considered. Occasional dependency
upgrades and CVEs might happen.

Thanks for using reveal-md! Please consider these Markdown-based alternatives:

- [MkSlides](https://martenbe.github.io/mkslides/) (successor to reveal-md)
- [Slidev](https://sli.dev)
